manifests/i/IrfanSkiljan/IrfanView-plugins/4.58/IrfanSkiljan.IrfanView-plugins.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: IrfanSkiljan.IrfanView-plugins
+PackageVersion: 4.58
+PackageLocale: en-US
+Publisher: Irfan Skiljan
+PublisherUrl: "https://www.irfanview.com/"
+PublisherSupportUrl: https://irfanview-forum.de
+PrivacyUrl: "https://www.irfanview.com/privacy_policy.htm"
+Author: Irfan Skiljan
+PackageName: IrfanView plugins
+PackageUrl: "https://www.irfanview.com/plugins.htm"
+License: Proprietary, free for non-commercial use
+LicenseUrl: "https://www.irfanview.com/eula.htm"
+Copyright: Copyright (C) 2021 by Irfan Skiljan, Wiener Neustadt, Austria.
+CopyrightUrl: https://www.irfanview.com/eula.htm
+ShortDescription: IrfanView plugins installer
+Description: IrfanView is an image viewer, editor, organiser and converter program for Microsoft Windows. The plugins provide additional filters, effects and file format support.
+Moniker: irfanview-plugins
+Tags:
+  - image
+  - image viewer
+  - photo
+  - picture
+  - graphic viewer
+  - graphic editor
```

Comment:
```suggestion
  - graphic-viewer
  - graphic-editor
```

Reply:
please note that these were copied from IrfanSkiljan.IrfanView, and should probably be changed there too if not appropriate

Reply:
The only reason why I did this was because of https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags.

**Note: The best practice is to present these terms in all lower case with hyphens rather than spaces.**